Makefile

```diff
@@ -24,7 +24,11 @@ endif
 
 .PHONY: build lint setup appimage
 
+ifeq ($(OS),linux)
```

Comment:
I think darwin build broke this the target on Linux, we have to specify appimage build dep